training/dictionary_pickler.py

```diff
@@ -140,13 +143,13 @@ def _pickle_dict(
         mydict = dict(sorted(mydict.items(), key=itemgetter(1)))
     if filepath is None:
         filename = f"strategies/dictionaries/data/{langcode}.plzma"
-        filepath = str(Path(simplemma.__file__).parent / filename)
+        filepath = str(Path(__file__).parent.parent / "simplemma" / filename)
```

Comment:
@juanjoDiaz I changed this line in order to store the pickled files locally, they need to be installed afterwards but I thought it's more logical.

Reply:
I just added a switch to make the change explicit.